packages/rich-text/package.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "@wordpress/rich-text",
+	"version": "1.0.0-beta.0",
+	"description": "Rich text value and manipulation API.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"rich-text"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/rich-text/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"main": "build/index.js",
+	"module": "build-module/index.js",
+	"react-native": "src/index",
+	"dependencies": {
+		"@babel/runtime": "^7.0.0",
+		"@wordpress/escape-html": "file:../escape-html",
```

Comment:
This introduces uncommitted changes to the local branch after `npm install`. Please review #10234

Reply:
Do you mean package lock? See #10304.